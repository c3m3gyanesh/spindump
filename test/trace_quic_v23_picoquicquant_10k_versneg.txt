Quant client vs. Picoquic server, using a fresh connection without any prior knowledge. Results in version negotiation between the two.
