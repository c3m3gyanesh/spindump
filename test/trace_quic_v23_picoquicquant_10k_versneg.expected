[
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "6aad60f365cdfc59732189041138-null (57924:4433)", "Ts": 1571088391317952, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "6aad60f365cdfc59732189041138-113822510b6aad (57924:4433)", "Ts": 1571088391394885, "State": "Starting", "Right_rtt": 76933, "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "null-113822510b6aad (57924:4433)", "Ts": 1571088391397507, "State": "Starting", "Left_rtt": 2622, "Packets1": 1, "Packets2": 1, "Bytes1": 1228, "Bytes2": 70, "Bandwidth1": 1228, "Bandwidth2": 70 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "null-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391475609, "State": "Up", "Right_rtt": 78102, "Packets1": 2, "Packets2": 1, "Bytes1": 2456, "Bytes2": 70, "Bandwidth1": 2456, "Bandwidth2": 70 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391560670, "State": "Up", "Transition": "0-1", "Who": "responder", "Packets1": 5, "Packets2": 6, "Bytes1": 2891, "Bytes2": 6734, "Bandwidth1": 2891, "Bandwidth2": 6734 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391637622, "State": "Up", "Transition": "1-0", "Who": "responder", "Packets1": 10, "Packets2": 13, "Bytes1": 3186, "Bytes2": 15761, "Bandwidth1": 3186, "Bandwidth2": 15761 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391637622, "State": "Up", "Full_rtt_responder": 76952, "Packets1": 10, "Packets2": 13, "Bytes1": 3186, "Bytes2": 15761, "Bandwidth1": 3186, "Bandwidth2": 15761 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391638694, "State": "Up", "Transition": "0-1", "Who": "initiator", "Packets1": 10, "Packets2": 14, "Bytes1": 3186, "Bytes2": 15828, "Bandwidth1": 3186, "Bandwidth2": 15828 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391638694, "State": "Up", "Left_rtt": 1072, "Packets1": 10, "Packets2": 14, "Bytes1": 3186, "Bytes2": 15828, "Bandwidth1": 3186, "Bandwidth2": 15828 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391647788, "State": "Up", "Transition": "0-1", "Who": "responder", "Packets1": 11, "Packets2": 14, "Bytes1": 3248, "Bytes2": 15828, "Bandwidth1": 3248, "Bandwidth2": 15828 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391647788, "State": "Up", "Right_rtt": 9094, "Packets1": 11, "Packets2": 14, "Bytes1": 3248, "Bytes2": 15828, "Bandwidth1": 3248, "Bandwidth2": 15828 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "5471c01a-08dcae6e04f5c86c (57924:4433)", "Ts": 1571088391647788, "State": "Up", "Full_rtt_responder": 10166, "Packets1": 11, "Packets2": 14, "Bytes1": 3248, "Bytes2": 15828, "Bandwidth1": 3248, "Bandwidth2": 15828 }
]
